runtime/browser/xwalk_presentation_service_delegate_android.h

```diff
@@ -1,37 +1,21 @@
-// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
 #ifndef XWALK_RUNTIME_BROWSER_XWALK_PRESENTATION_SERVICE_DELEGATE_ANDROID_H_
 #define XWALK_RUNTIME_BROWSER_XWALK_PRESENTATION_SERVICE_DELEGATE_ANDROID_H_
 
 #include <string>
-#include <utility>
-
-#include "base/containers/scoped_ptr_hash_map.h"
-#include "base/memory/scoped_ptr.h"
-#include "base/memory/weak_ptr.h"
-#include "content/public/browser/presentation_service_delegate.h"
-#include "content/public/browser/web_contents_user_data.h"
-
-namespace content {
-class WebContents;
-}  // namespace content
+#include "xwalk/runtime/browser/xwalk_presentation_service_delegate.h"
 
 namespace xwalk {
 
-class PresentationFrame;
-class PresentationSession;
-class XWalkBrowserContext;
-
 class XWalkPresentationServiceDelegateAndroid
-    : public content::WebContentsUserData
-        <XWalkPresentationServiceDelegateAndroid>,
-      public content::PresentationServiceDelegate,
+    : public XWalkPresentationServiceDelegate,
+      public content::WebContentsUserData<
+        XWalkPresentationServiceDelegateAndroid>,
```

Comment:
Hi @rakuco, thanks very much for the info. This was really pain in the a** and now most of the format issues are gone. very useful, totally :)